packages/wallet-adapter-core/src/sdkWallets.ts

```diff
@@ -36,7 +35,7 @@ export function getSDKWallets(dappConfig?: DappConfig) {
           network: dappConfig.network as any,
           manifestURL: dappConfig.mizuwallet.manifestURL,
           appId: dappConfig.mizuwallet.appId,
-        }) as any,
+        }) as any
```

Comment:
is this cast needed? is `MizuWallet` non-compliant?

Reply:
It wasn't at that time, let me check again